apps/asap-server/test/handlers/users/fetch.test.ts

```diff
@@ -52,44 +50,10 @@ describe('GET /users', () => {
     nock(cms.baseUrl)
       .get(`/api/content/${cms.appName}/users`)
       .query({
-        q: JSON.stringify({
-          take: 8,
-          filter: {
-            or: [
-              {
-                path: 'data.displayName.iv',
-                op: 'contains',
-                value: 'first',
-              },
-              {
-                path: 'data.firstName.iv',
-                op: 'contains',
-                value: 'first',
-              },
-              {
-                path: 'data.lastName.iv',
-                op: 'contains',
-                value: 'first',
-              },
-              {
-                path: 'data.displayName.iv',
-                op: 'contains',
-                value: 'last',
-              },
-              {
-                path: 'data.firstName.iv',
-                op: 'contains',
-                value: 'last',
-              },
-              {
-                path: 'data.lastName.iv',
-                op: 'contains',
-                value: 'last',
-              },
-            ],
-          },
-          sort: [{ path: 'data.displayName.iv' }],
-        }),
+        $top: 8,
+        $orderby: 'data/displayName/iv',
+        $filter:
+          "data/teams/iv/role eq 'Lead PI' and (contains(data/displayName/iv, 'first') or contains(data/firstName/iv, 'first') or contains(data/displayName/iv, 'last') or contains(data/firstName/iv, 'last'))",
```

Comment:
Can we keep this under 80 char long?

Reply:
Not really, but why is that? I thought we had seen that the max query string length was 1024 chars. This on right here has ~200 chars

Reply:
80 char long line, to be more readable 😛